<ui version="4.0" >
 <class>choose_dl_form</class>
 <widget class="QDialog" name="choose_dl_form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Выбор текущего должностного лица</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Вы назначены на несколько должностей. Выберите текущую:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QTreeWidget" name="twDls" >
     <column>
      <property name="text" >
       <string>Должностное лицо</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Должность</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Подразделение</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel" >
       <property name="text" >
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
